src/libs/models/BankAccount.js

```diff
@@ -279,47 +279,6 @@ class BankAccount {
     needsToUpgrade() {
         return !this.isInSetup() && !lodashHas(this.json, ['additionalData', 'beneficialOwners']);
     }
-
-    /**
-     * Check if we've performed the most recently implemented checks on the bank account, and they all passed.
-     * Same logic as in BankAccount.php needsToPassLatestChecks
-     * @return {Boolean}
-     */
-    needsToPassLatestChecks() {
```

Comment:
Looks like we already have this logic in Web-E so that's why we don't need it duplicated here, correct?

Reply:
Yeah, basically this was used when we added the beneficialOwners step to the flow, so all existing bank accounts can upgrade. But most of them upgraded, and App doesn't actually provide the "upgrade" flow, so this is some code complexity that we can remove and that allows us not to have to rely on externalApiResponses from App anymore.